tests/cases/fourslash/refactorConvertToGetAccessAndSetAccess1.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+//// class A {
+////     /*a*/public a: string;/*b*/
+//// }
+
+goTo.select("a", "b");
```

Comment:
I believe we have a range API instead, but I haven't ever written one of these tests yet, so this *might* be fine?

Reply:
i'm not sure...
I copied it from the refactor ConverT Es6 Module